pkg/probe/probes.go

```diff
@@ -127,25 +134,31 @@ func checkNodeReadiness(cli client.Client) (bool, error) {
 	return false, nil
 }
 
-func defaultGw() (string, error) {
-	gjsonCurrentState, err := currentStateAsGJson()
-	if err != nil {
-		return "", errors.Wrap(err, "failed retrieving current state to retrieve default gw")
-	}
-	defaultGwGjsonPath := "routes.running.#(destination==\"0.0.0.0/0\").next-hop-address"
-	defaultGw := gjsonCurrentState.Get(defaultGwGjsonPath).String()
-	if defaultGw == "" {
+func defaultGw(currentState gjson.Result) (string, error) {
+	var found gjson.Result
+	currentState.Get("routes.running").ForEach(
+		func(_, v gjson.Result) bool {
+			// we want to pick the next hop related to the "main" table because we may have multiple tables
+			if v.Get("destination").String() == "0.0.0.0/0" && v.Get("table-id").Int() == 254 {
+				found = v.Get("next-hop-address")
+				return false
+			}
+			return true
+		},
+	)
+
+	if !found.Exists() {
 		msg := "default gw missing"
-		defaultGwLog := log.WithValues("path", defaultGwGjsonPath)
+		defaultGwLog := log.WithValues("path", "routes.running.next-hop-address")
```

Comment:
Can we add the table-id as part of the log ?

Reply:
done